### ISSUE TITLE
All the requests are submitted as admin request

### ISSUE THREAD

**Issue by rshamsz** [StackTraceYo/plex-bot#35]
I have configured BOT_ADMIN to authorize users to submit their requests, but all the requests go through and submitted by the Ombi admin users as defined in the env file; as a result, the Ombi user Quota does not get applied to the users and also Ombi notifications all go to the admin user.
Is this an expected behavior for now? and is there any plan to make user request management available.
I know that the similar bots in Discord can do user request management and correlated Discoed users to Ombi users.

Thanks for your effort here, looks very promising 👍

**Comment by StackTraceYo** [comment]
so wed have to register users with their ombi name?

**Comment by rshamsz** [comment]
not exactly; each OMBI user can be configured with a Telegram ID; I was just wondering if it is possible that:   
When a user puts a request in a Telegram group can be correlated to the OMBI user which has configured with same Telegram ID as the user who is putting the request and as a result, the quota of that user being used. The other advantages are 
1- Ombi admin can see who has put the request on Ombi Panel.
2- user can be notified when 1- the request has been approved 2-the requested has been downloaded


Thanks
